server.ts

```diff
@@ -1,15 +1,24 @@
 import * as bodyParser from 'body-parser';
 import * as express from 'express';
-import { graphqlExpress } from 'apollo-server-express';
 import { getPortPromise } from 'portfinder';
 import { schema } from './graphql';
 import * as cors from 'cors';
+var graphqlHTTP = require('express-graphql');
+import * as jwtExpress from 'jwt-express';
 
 const app = express();
-
 app.use(cors());
 
-app.use('/graphql', bodyParser.json(), graphqlExpress({ schema }));
+app.use(
+  '/graphql',
+  bodyParser.json(),
+  jwtExpress.init('secret', { cookies: false }),
```

Comment:
Would this support setting the secret at run time, or verifying a signature via public key? Or is that a step we put in the middleware to set the value for jwt.valid?

Reply:
It doesn't support promises, so I'm thinking once we have the endpoint we might want to just write our own middleware and use the functions you wrote. 

Or we can make the network request to get the token before initializing express.

Reply:
Good idea to work within existing systems as much as possible. Let's look deeper into this